draft-ietf-tls-tls13.md

```diff
@@ -3429,6 +3431,35 @@ flight, the server MUST be prepared to receive an arbitrary number
 of such messages before receiving the Authentication messages.
 
 
+#### Key Update 
+
+struct {} KeyUpdate;
+
+The KeyUpdate handshake message is used to indicate that the sender is
+updating its sending cryptographic keys. This message can be sent by
+the server after sending its first flight and the client after sending
+its second flight. Implementations prior to this time MUST generate a
+fatal "unexpected_message" alert.  After sending a KeyUpdate, the
+sender SHALL send all its traffic using the next generation of keys,
+computed as described in {{updating-traffic-keys}}. Upon receiving a
+KeyUpdate, the receiver MUST update their receiving keys and if they
+have not already updated their sending state up to or past the then
+current receiving generation MUST immediately send their own
```

Comment:
What does "immediately" mean, and what are the consequences if the receiver does not actually send this KeyUpdate?  If there is no KeyUpdate should the original sender send an alert and close the connection?  If so, under what conditions?